python/core/memory-allocation/README.md

```diff
@@ -1,12 +1,15 @@
 name: Memory Allocation
+
 description: On how memory allocation and de-allocation works.
+
 insights:
+  - the-is-operator
```

Comment:
lgtm, but wouldn't it be better if we first introduce pointers and references then use them as explanation for this insight?

Reply:
yes, that would be better :+1: